src/openzaak/components/zaken/tests/test_notifications_send.py

```diff
@@ -1124,6 +1126,135 @@ def test_send_notif_update_zaak_eigenschap(self, mock_notif):
             },
         )
 
+    @override_settings(ENABLE_CLOUD_EVENTS=True)
+    @patch("openzaak.components.zaken.api.cloud_events.CloudEventConfig.get_solo")
```

Comment:
Could you move these tests to its own file `test_cloud_events.py`?

You can use the `@patch` decorator at the class level and then configure it immediately:

```suggestion
    @patch("openzaak.components.zaken.api.cloud_events.CloudEventConfig.get_solo", return_value=CloudEventConfig(enabled=True, source="...", ...))
```